.github/workflows/gradle.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Gradle
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        java: [1.8, 11]
```

Comment:
Heh. It occurs to me that this would almost be better as a global variable so that when we add versions we don't need to touch every action.

Reply:
This is coming from the template. I will open an issue
the same will apply to Python version